tests/geobase/checks_FairGeoBasicShape.h

```diff
@@ -0,0 +1,161 @@
+/********************************************************************************
+ *   Copyright (C) 2024 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH     *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+
+#include "FairGeoBasicShape.h"
+#include "FairGeoRotation.h"
+#include "FairGeoTransform.h"
+#include "FairGeoVector.h"
+#include "FairGeoVolume.h"
+#include "generate_TestInput.h"
+
+#include <TArrayD.h>
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/matchers/catch_matchers_floating_point.hpp>
+#include <string>
+
+using Catch::Matchers::WithinRel;
+
+#ifndef TESTS_GEOBASE_CHECKS_FAIRGEOBASICSHAPE_H
+#define TESTS_GEOBASE_CHECKS_FAIRGEOBASICSHAPE_H
+
+namespace fairroot::tests
+{
+
+auto check_FairGeoTransform(FairGeoTransform* _trans)
```

Comment:
Why not pass by reference? You expect it to be non-nullptr anyway?

```suggestion
auto check_FairGeoTransform(FairGeoTransform& _trans)
```

Reply:
Done.